src/components/ColourStyles.js

```diff
@@ -0,0 +1,50 @@
+import chroma from 'chroma-js';
+
+const ColourStyles = {
```

Comment:
Can you add some comments explaining each method, and how each one modifies the styles?

Reply:
Yeah!